### DISCUSSION TITLE
Using Brave browser for PDF export?

### DISCUSSION THREAD

**Discussion by travisfranck** [marp-team/marp#80] [Q&A] [answered]
I'm running macOS Big Sur on an intel MacBook Pro.

The only chromium-based browser I have installed it Brave.  

I've looked through the app bundle contents and can't find the correct binary to link to.  Can I use Brave to export my slides from VSCode to a PDF?

TIA

**Comment by yhatt** [comment, marked as answer]
Yes, you can. The most of Chromium-flavored browsers for macOS have an executable binary in `XXXXXX.app/Contents/MacOS/XXXXXX`, and you can find out it from `Show Package Contents` in the context menu of application.

In the case of Brave, there is in `/Applications/Brave Browser.app/Contents/MacOS/Brave Browser`. Please set the path to `Markdown › Marp: Chrome Path` in the extension setting.

**Comment by travisfranck** [comment]
Worked great!  Thank you.

I entered I had tried that same binary, but forgot to include the .app extension on the bundle, so I had the path wrong.
